javalib/src/main/scala/java/util/regex/UnicodeTables.scala

```diff
@@ -0,0 +1,5051 @@
+// AUTOGENERATED by make_unicode_tables.awk from the output of
+// go/src/pkg/unicode/maketables.go.  Yes it's awful, but frankly
+// it's quicker than porting 1300 more lines of Go.
+
+package java.util.regex
+
+import java.util.HashMap
+import java.util.Map
+
+object UnicodeTables {
```

Comment:
At the moment our plugin can't handle this as constructor of the module is essentially a single 5k LOC method. We've not seen methods this big before.